bin/minimalcss.js

```diff
@@ -73,6 +74,20 @@ const options = {
   urls: urls,
   debug: argv['debug'],
   loadimages: argv['loadimages'],
+  skippable: request => {
+    let skips = argv['skip']
+    if (!skips) {
+      return false
+    }
+    if (!Array.isArray(skips)) {
+      skips = [skips]
+    }
+    return skips.some(skip => {
+      return request.url.match(
+        new RegExp(skip.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, '\\$1'), 'i')
```

Comment:
is this for domain extraction? some comment or tests would be helpful

Reply:
I'm glad I asked for a review. No that's just silly to do now. To do case sensitive string comparison we can simply do `url.toLowerCase().indexOf(skip.toLowerCase()) > -1`
Thanks for pointing that out!